tests/commands/test_preview_module.py

```
@@ -0,0 +1,602 @@
+"""Integration tests for the preview_module command."""
+
+import pytest
+import os
+import tempfile
+import yaml
+import click
+from click.testing import CliRunner
+from unittest.mock import patch
+
+from ftf_cli.commands.preview_module import preview_module
+from ftf_cli.operations import ModuleOperationError, register_module, publish_module
```

Ruff (0.11.9)

`ftf_cli.operations.register_module` imported but unused

Remove unused import

(F401)